app/services/katello/product_content_importer.rb

```diff
@@ -20,11 +20,13 @@ class ProductContentImporter
     #    },
     #    "enabled":false
     # }
+    attr_accessor :content_url_updated
```

Comment:
I think what you want here is an attr_reader since this isn't something we want to adjust from the outside world
```
attr_reader :content_url_updated
```
in initialize:

```
@content_url_updated = []
```

in update_content

```
content_url_updated << content
```